src/main/java/core/basesyntax/model/Operation.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.model;
+
+public enum Operation {
+    BALANCE("b"),
+    SUPPLY("s"),
+    PURCHASE("p"),
+    RETURN("r");
+
+    private String code;
```

Comment:
According to the encapsulation principle, the field 'code' should be private to hide the internal representation of the enum.